index.html

```diff
@@ -194,7 +194,7 @@ <h2 class="wow slideInLeft" data-wow-duration=".5s"><span>Our</span> Sponsors</h
     </section>
     <section class="speaker_sec bg_greyclr" id="speakerID">
 	<div class="container">
-		<!--
+		
```

Comment:
The "There will be two kinds of talks" section is wrong. We have regular and lighting this year.

Reply:
Also two tracks